packages/browser-wallet/src/popup/shared/account-info-listener.ts

```diff
@@ -23,7 +23,30 @@ export class AccountInfoListener extends EventEmitter {
         this.genesisHash = network.genesisHash;
     }
 
+    private async update(accountAddress: string, blockHash?: string) {
+        try {
+            if (this.accountsMap.size > 0) {
```

Comment:
I do not think this check should be in here. (Why did you move it inside the for loop?) :thinking: